### PR TITLE
Add content/en/database as a service

### PR DESCRIPTION
**Pull request by Asmit2952**
Added a new term: Database as a Service (DBaaS)
Fixes: #340

### CODE REVIEW

File: content/en/database_as_a_service.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+category: Technology
+---
+
+## What it is
+
+Database-as-a-Service (DBaaS) is a service managed by a cloud operator (public or private) that supports applications without requiring the application team to perform traditional database administration functions. With DBaaS, application developers should not need to be database experts, nor should they need to hire a database administrator (DBA) to keep the database up to date.
+
+## Problem it addresses 
+
+Traditionally in on-premise setups, organizations have to invest in additional storage and processing capacity on a regular basis to accommodate database expansion. Developers also need to provision and configure database with the help of IT infrastructure team. Deployment speed and database driven applications takes longer time to load and execute. This lack of features and scalibilty leads to increased costs and slow applications.
```
Comment (jihoon-seo) [inline content/en/database_as_a_service.md:13]: Please change "on-premise" to "on-premises". 😃

**Review by jihoon-seo — COMMENTED** [review]
@Asmit2952 Thank you for the contribution. 😃
PTAL!

File: content/en/database_as_a_service.md (hunk 1 above)
Comment (Asmit2952) [reply content/en/database_as_a_service.md]: Done!!

Comment (CathPag) [reply content/en/database_as_a_service.md]: Here also some edits with a few tweaks:

> Traditionally, in on-premise setups, organizations regularly have to invest in additional storage and processing capacity to accommodate database expansion which can be expensive.
> 
> Additionally, developers typically provision and configure databases with the help of IT infrastructure teams, slowing deployment speed of database-driven applications down. Loading and executing them also takes longer.

File: content/en/database_as_a_service.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## How it helps
+
+With DBaaS, it is possible to manage diverse sets of database from a single platform. Once a database is deployed, the cloud based service provider, is in charge of all backend operations to keep it running smoothly, including configuration management, backups, patches, upgrades, service monitoring, and more, with a user-friendly interface to manage it all. This helps organizations to develop enterprise level applications faster and keep if affordable at same time.
```
Comment (CathPag) [inline content/en/database_as_a_service.md:17]: Here are some edits. As to the content, I'll need a technical reviewer to look at it as well. 

> DBaaS allows users to manage databases from a single platform, while the service provider handles all backend operations. The DBaaS provider ensures the database runs smoothly, takes care of all configuration management, backups, patches, upgrades, service monitoring, and provides a user-friendly interface. This helps organizations develop enterprise-grade applications faster while minimizing database costs.

File: content/en/database_as_a_service.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Database as a Service (DBaaS)
+status: Completed
```
[the remark this answers is not in the document]
Comment (JasonMorgan) [reply content/en/database_as_a_service.md:3]: I think this is close enough that we should be able to merge it with only a few changes.

File: content/en/database_as_a_service.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+category: Technology
+---
+
+## What it is
+
+Database-as-a-Service (DBaaS) is a service managed by a cloud operator (public or private) that supports applications without requiring the application team to perform traditional database administration functions. With DBaaS, application developers should not need to be database experts, nor should they need to hire a database administrator (DBA) to keep the database up to date.
```
[the remark this answers is not in the document]
Comment (JasonMorgan) [reply content/en/database_as_a_service.md:9]: I think this is fine either way. I'm not entirely convinced that using a DBaaS means you won't need a DBA but it certainly means fewer DBAs can do more.

**Comment by CathPag** [comment]
Oh, and spellcheck is failing. Can you please make sure that's fixed as well? Thank you!

File: content/en/database_as_a_service.md (hunk 2 above)
Comment (JasonMorgan) [reply content/en/database_as_a_service.md:17]: I don't quite get the single platform line and I'm not sure it adds a ton to the definition. The tl;dr on a DBaaS is you get the database API without needing to do most of the operations and management of a database itself.

Comment (JasonMorgan) [reply content/en/database_as_a_service.md:17]: Overall the content looks amazing. Great work @Asmit2952!

File: content/en/database_as_a_service.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Asmit2952) [reply content/en/database_as_a_service.md:9]: I agree, but what I think is, even a full stack developer can handle DBaaS very well without getting into the complex stuff

File: content/en/database_as_a_service.md (hunk 2 above)
Comment (Asmit2952) [reply content/en/database_as_a_service.md:17]: > Overall the content looks amazing. Great work @Asmit2952!

Thank you!!

**Comment by Asmit2952** [comment]
> Oh, and spellcheck is failing. Can you please make sure that's fixed as well? Thank you!

What do I correct in words like DBaaS/DBA?

File: content/en/database_as_a_service.md (hunk 2 above)
Comment (iamNoah1) [reply content/en/database_as_a_service.md]: Thanks @Asmit, I agree the content looks good. I would remove the single platform thing like Jason suggested and also replace "backend" with "administration/administrative".

**Review by iamNoah1 — COMMENTED** [review]
Special words for the spellchecker have to be added to the wordlist file: https://github.com/cncf/glossary/blob/b60ee9dbc4ccf04a11e6c4f1ef1ed6d05779d17c/wordlist.txt

File: content/en/database_as_a_service.md (hunk 2 above)
Comment (Asmit2952) [reply content/en/database_as_a_service.md]: Okay, I'll do that

Comment (Asmit2952) [reply content/en/database_as_a_service.md]: Do I completely remove single platform thing, because cloud platform looks fine

Comment (Asmit2952) [reply content/en/database_as_a_service.md]: Okay then, I'll remove that line

**Comment by CathPag** [comment]
Edited the first sentence of how it helps

**Review by iamNoah1 — APPROVED** [review]
lgtm

**Review by CathPag — APPROVED** [review]
Looks great, thank you!

**Pull request merged**